administrator/modules/mod_latest/src/Helper/LatestHelper.php

```diff
@@ -38,7 +38,7 @@ abstract class LatestHelper
      */
     public static function getList(Registry $params, ArticlesModel $model)
     {
-        $user = Factory::getUser();
+        $user  = Factory::getApplication()->getIdentity();
```

Comment:
```suggestion
        $user = Factory::getApplication()->getIdentity();
```
Remove redundant space.